frame/nft/src/lib.rs

```diff
@@ -224,7 +226,9 @@ pub mod pallet {
 							Some(instances) => {
 								instances.remove(&(*class, *instance));
 							},
-							None => {},
+							None => {
+								debug_assert!(false, "unreachable")
```

Comment:
i wold prefer if let with such empty map. or panic!.

Reply:
Will review this when reviewing ValueQuery